app/backend/src/couchers/servicers/api.py

```diff
@@ -517,9 +517,6 @@ def UpdateProfile(self, request, context, session):
             else:
                 user.camping_ok = request.camping_ok.value
 
-        # save updates
-        session.commit()
-
```

Comment:
Why remove this?

Reply:
It's not needed, when it leaves the `session_scope` it's wrapped in, it automatically commits the transaction. It's form a bygone era :P